File: mvp.admin/src/FlexForm.Api.Admin/Model/FormControlItemSourceModel.cs
namespace FlexForm.Api.Admin.Model
{
    public class FormControlItemSourceModel : ModelBaseInt
    {
        public string ItemSourceCondition { get; set; }

        public string ItemSourceOrder { get; set; }

        public string IncludeProperties { get; set; }

        public int PageSize { get; set; }
    }
}

File: mvp.admin/src/FlexForm.Admin/flexform-admin/src/app/models/dtos/form-control-item-source.model.ts
import { Guid } from 'guid-typescript';
import { BaseWithTimeStamp } from './base-with-timestamp.model';

export class FormControlItemSource extends BaseWithTimeStamp<number>{

    guid:Guid;

    constructor() {
        super();
        this.guid = Guid.create();
    }

    pageSize:Number;
    itemSourceCondition: string;
    itemSourceOrder: string;
    includeProperties: string;
}